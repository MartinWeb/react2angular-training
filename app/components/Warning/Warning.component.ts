import angular from "angular";
import template from './warning-template.html';

export const WarningComponent: angular.IComponentOptions = {
    template,
};