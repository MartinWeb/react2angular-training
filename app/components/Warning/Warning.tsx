import { Alert } from "components/Alert/Alert";

export const Warning = () => (
    <Alert type="warning" title="Be very careful not to mock this website's design !">
        <p>Chuck loves orange and blue, because it's the colour of his beard and his favorite jeans.</p>
    </Alert>
);
