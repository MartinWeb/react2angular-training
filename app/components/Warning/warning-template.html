<alert-component title="Be careful, Chuck Norris is so strong, even browsing here could destroy you !" type="warning">
    <p>You have benn warned my friend :D</p>
</alert-component>