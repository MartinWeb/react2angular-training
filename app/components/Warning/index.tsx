import { react2angular } from '@martinweb/react2angular';
import { Warning } from './Warning';

const WarningComponent = react2angular(() => <Warning />);

export { WarningComponent, Warning };
