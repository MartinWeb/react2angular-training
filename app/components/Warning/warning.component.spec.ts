import { WarningComponent } from "./Warning.component";

describe('WarningComponent', () => {
    test('should have valid snapshot', () => {
        expect(WarningComponent.template).toMatchSnapshot();
    });
});