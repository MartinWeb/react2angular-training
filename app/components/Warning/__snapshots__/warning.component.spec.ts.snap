// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningComponent should have valid snapshot 1`] = `
"<alert-component title="Be very careful not to mock this website's design !" type="warning">
    <p>Chuck loves orange and blue, because it's the colour of his beard and his favorite jeans.</p>
</alert-component>"
`;
